test/aleph/http_test.clj

```diff
@@ -570,3 +570,40 @@
                                       (.write ctx msg p)))))})
       (let [resp @(http-get (str "http://localhost:" port "/string"))]
         (is (= test-header-val (get (:headers resp) test-header-name)))))))
+
+(defn add-http-object-aggregator [^ChannelPipeline pipeline]
+  (.addBefore pipeline
+              "request-handler"
+              "http-object-aggregator"
+              (HttpObjectAggregator. 5)))
```

Comment:
Can we document that this means the max length is 5 bytes, either via docstring or replacing the magic num witha var?

Reply:
I guess it's good enough like this:

https://github.com/clj-commons/aleph/pull/484/commits/5d2689218eb603fec67bea1e08999a1c403608bd#diff-b7e83256bff42d68457008e51789109ff14c053e0d6de64c6ffa03d1d11b3dcfR575